cmd/keycloakb/keycloak_bridge.go

```diff
@@ -775,13 +775,17 @@ func config(logger log.Logger) *viper.Viper {
 	// Storage events in DB (read/write)
 	v.SetDefault("events-db", false)
 	configureDbDefault(v, "db-audit-rw")
+	v.SetDefault("db-audit-migration", false)
+	v.SetDefault("db-audit-migration-version", "")
```

Comment:
What if xxx-migration-version is empty and xxx-migration is true ? Should the second parameter be automatically considered as false ?

Reply:
Very good point: this should not happen; but if it is the case and the version number is missing we cannot do the verification in any case. So, I would answer to your second question with yes: if no version number is given then the check should not be done.
Do you agree?

Reply:
migration=true + migration-version empty:
- we consider that the check is disabled
- we consider that it is an error: the application stops
You can choose... ;-)